### DISCUSSION TITLE
Add VLC Player (Use local device as a media player)

### DISCUSSION THREAD

**Discussion by Surgical17** [music-assistant/support#1814] [Feature requests and ideas]
Hi There

Please add the VLC player to use the local device where Home Assistant is running as a media player:|
![image](https://github.com/music-assistant/hass-music-assistant/assets/76941635/c46a1a87-f497-408e-8b50-78732dd0ea1f)

This feature/option did work on the original Music Assistant Add-On.

**Comment by OzGav** [comment]
Please don't open duplicate discussions https://github.com/orgs/music-assistant/discussions/1234
